src/app/LoginController.java

```diff
@@ -41,6 +46,10 @@
 
    private Timer timer;
 
+    public LoginController() throws AccountDaoException {
```

Comment:
This is a fatal exception and we should catch it instead of throwing it.. We can just log it and make the login view show the error message.